sdk/communication/Azure.Communication.CallingServer/src/AmsDirectRequestHelpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    internal static class AmsDirectRequestHelpers
+    {
+        internal static HttpMessage GetHttpMessage(CallingServerClient client, Uri requestEndpoint, RequestMethod requestMethod, HttpRange? rangeHeader = null)
+        {
+            Argument.CheckNotNull(client, nameof(client));
+            Argument.CheckNotNull(requestEndpoint, nameof(requestEndpoint));
+            Argument.CheckNotNullOrEmpty(requestMethod.Method, nameof(requestMethod));
+
+            HttpMessage message = client._pipeline.CreateMessage();
+            Request request = message.Request;
+            request.Method = requestMethod;
+            var uri = new RequestUriBuilder();
```

Comment:
nit. You should be consistent in your usage of var. Since you are qualifying type for the rest of your variables, you should also do so here.

Reply:
not blocking.